app/views/admin/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %h1.govuk-heading-xl
+      Add a new user
+    .ccs-user-form-group
+      = form_for [:admin, @user] do |form|
+        - @user.errors.full_messages.each do |message|
+          = message
```

Comment:
Once you've updated the fields below, you should be able to create some error messages, which will need to be styled. One error message is if you try and create a user with an email address that already exists.